libs/sdk-ui-ext/src/internal/utils/bucketHelper.ts

```diff
@@ -350,11 +351,11 @@ export function setBucketTitles(
 }
 
 export function generateBucketTitleId(localIdentifier: string, visualizationType: string): string {
-    return `dashboard.bucket.${localIdentifier}_title.${visualizationType}`;
+    return titles[`${localIdentifier}_${visualizationType}`].id;
```

Comment:
This works with the localisation validation?

Reply:
Yes, because messages are defined using `defineMessages` and this is only call to map :)